src/test/java/elizCurtnazar/tests/DemoQaManager.java

```diff
@@ -0,0 +1,167 @@
+        radioButton.click();
+        WebElement question = driver.findElement(By.cssSelector(".mb-3"));
+        question.getText();
+        System.out.println(question.getText());
+        WebElement yesAnswer = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[2]/label"));
+        yesAnswer.click();
+        WebElement answer = driver.findElement(By.cssSelector(".mt-3"));
+        answer.getText();
+        System.out.println(answer.getText());
+    }
+
+
+    public void WebTablesLink(String link) {
+
+        String urlBase = "https://demoqa.com/";
+        driver.get(urlBase + link);
+
+
+        WebElement addButton = driver.findElement(By.id("addNewRecordButton"));
+        addButton.click();
+        WebElement firstNameInput = driver.findElement(By.id("firstName"));
+        firstNameInput.click();
+        firstNameInput.clear();
+        firstNameInput.sendKeys(firstName);
+        WebElement lastNameInput = driver.findElement(By.id("lastName"));
+        lastNameInput.click();
+        lastNameInput.clear();
+        lastNameInput.sendKeys(lastName);
+        WebElement emailInput = driver.findElement(By.id("userEmail"));
+        emailInput.click();
+        emailInput.clear();
+        emailInput.sendKeys(email);
+        WebElement ageInput = driver.findElement(By.id("age"));
+        ageInput.click();
+        ageInput.clear();
+        ageInput.sendKeys(age);  // cum aduag in sendkeys un int?
+        WebElement salaryInput = driver.findElement(By.id("salary"));
+        salaryInput.click();
+        salaryInput.clear();
+        salaryInput.sendKeys(salary);
+        WebElement departmentInput = driver.findElement(By.id("department"));
+        departmentInput.click();
+        departmentInput.clear();
+        departmentInput.sendKeys(department);
+        WebElement submitClick = driver.findElement(By.id("submit"));
+        submitClick.click();
+
+    }
+
+
+    public void ButtonsLink(String link) {
+        String urlBase = "https://demoqa.com/";
+        driver.get(urlBase + link);
+
+        WebElement doubleClickMe = driver.findElement(By.id("doubleClickBtn"));
+        doubleClickMe.click();
+
+    }
+
+    public WebElement LinkElementLink(String link) {
+        String urlBase = "https://demoqa.com/";
+        driver.get(urlBase + link);
+
+        List<WebElement> linkList = driver.findElements(By.id("linkWrapper"));
+        List<String> list = new ArrayList<>();
+        for (WebElement elem : linkList) {
+            list.add(elem.getText());
+        }
+        System.out.println(list);
+        WebElement homeLink = driver.findElement(By.id("simpleLink"));
+        homeLink.click();
+        return homeLink;
+
+
+    }
+}
+//       WebElement webTable = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[4]"));
```

Comment:
Why are these lines commented? Aren't they used anymore? If so, please remove them. Thanks!

Reply:
Sunt unele varianta pe care vroiam sa le fac. Pot sa le las sau trebuie neaparat sterse?

Reply:
In mod normal, s-ar sterge. Insa ai putea sa lasi un comentariu in care sa spui ca sunt alte variante pe care le-ai incercat sau ceva de genul si ca le pastrezi pentru istoric.